source/Calamari/Kubernetes/ManifestReporter.cs

```diff
@@ -39,9 +39,6 @@ public ManifestReporter(IVariables variables, ICalamariFileSystem fileSystem, IL
 
         public void ReportManifestFileApplied(string filePath)
         {
-            if (!FeatureToggle.KubernetesLiveObjectStatusFeatureToggle.IsEnabled(variables)
-                && !OctopusFeatureToggles.KubernetesObjectManifestInspectionFeatureToggle.IsEnabled(variables))
-                return;
```

Comment:
as MI feature toggle is `true`, this is _always_ false, so the check can be removed